### PR TITLE
added GlowlessScrollBehavior

### LINKED ISSUE #76490: add GlowlessScrollBehavior
it would be a handly class to easily disable scroll glow without need for searching up "disable scroll glow flutter", which is by the way a very popular question on SO https://stackoverflow.com/questions/51119795/how-to-remove-scroll-glow

### PR DESCRIPTION
**Pull request by nt4f04uNd**
fixes https://github.com/flutter/flutter/issues/76490

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I signed the [CLA].
- [ ] All existing and new tests are passing.

### CODE REVIEW

File: packages/flutter/test/widgets/scroll_behavior_test.dart [hunk 1]
```diff
@@ -32,13 +32,25 @@ class TestScrollBehavior extends ScrollBehavior {
...
+    expect(find.byType(GlowingOverscrollIndicator), findsNothing);
+  });
+
   testWidgets('Inherited ScrollConfiguration changed', (WidgetTester tester) async {
-    final GlobalKey key = GlobalKey(debugLabel: 'scrollable');
     TestScrollBehavior? behavior;
     late ScrollPositionWithSingleContext position;
 
     final Widget scrollView = SingleChildScrollView(
-      key: key,
```
Comment (nt4f04uNd) [inline packages/flutter/test/widgets/scroll_behavior_test.dart:41]: this key was pointless i guess

**Review by Piinks — COMMENTED** [review]
Hi @nt4f04uNd thanks for the contribution!
I am not sure this is the best way to solve the issue you filed. I agree we need to make it easier for users to modify the `ScrollConfiguration` and `ScrollBehavior`, that's actually something I am working on right now. :)
Adding a `GlowlessScrollBehavior` is a really specific solution that is not immediately clear how to use or implement. Like in the SO question you shared in #76490, it is not very discoverable how to modify the ScrollConfiguration, so even with this here I don't know that folks will know to use it.
I am working on a proposal now to expose all of this to users so it is more accessible, I'll share it once I have it ready. I would love to have your feedback on it.

**Comment by nt4f04uNd** [comment]
@Piinks ok, please cc me please when it's ready. i agree this can and should be more explorable

**Comment by goderbauer** [comment]
I am closing this based on the discussion above.

**Pull request closed without merging**